lib/stripe/api.ex

```diff
@@ -25,7 +25,7 @@ defmodule Stripe.API do
       json_library: Jason # defaults to Poison but can be configured to Jason
```

Comment:
Could we also take care of `http_module` module attribute and turn that into a function here? https://github.com/code-corps/stripity_stripe/blob/0493246373c80e3984644e1369528f6de4ed3d6e/lib/stripe/api.ex#L20

Reply:
Implemented in `6fdf9b3`